packages/common/src/core/types/DynamicValues.ts

```diff
@@ -0,0 +1,102 @@
+import { ISearchOptions } from "@esri/arcgis-rest-portal";
```

Comment:
Since that's from rest-js, VSCode will show some minimal info inline, and F12 will open the type definition file. A comment on this line won't go into the hub.js docs. I just added a link to the rest-js doc from the `ItemSearchOptions` type defintion doc string